plugins/woocommerce-admin/client/marketplace/utils/tracking.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+
+interface MarketplaceViewProps {
+	view: string;
+	searchTerm?: string;
+	productType?: string;
+	category?: string;
+}
+
+interface LegacyTabViewProps {
+	view: string;
+	search_term?: string;
+	category?: string;
+}
+
+/**
+ * Record a marketplace view event.
+ * This is a new event that is easier to understand and implement consistently
+ */
+function recordMarketplaceView( props: MarketplaceViewProps ) {
+	const eventProps = { ...props };
+
+	recordEvent( 'marketplace_v2_view', eventProps );
```

Comment:
I would be comfortable with calling this `marketplace_view`, as it won't be too hard to find out when we introduced it.